File: tests/language_completion/fixtures/cpp/functions_conditionals.cpp
#include <iostream>
#include <string>

int score(int value)
{
    if (value > 5)
    {
        return value + 1;
    }
    return value + 2;
}

int main()
{
    int result = score(5);
    std::string label = "small";
    if (score(8) > 7)
    {
        label = "big";
    }
    std::cout << "cpp_branch " << label << " " << result << std::endl;
}

File: tests/language_completion/fixtures/csharp/functions_conditionals.cs
using System;

public class Program
{
    public static int Score(int value)
    {
        if (value > 5)
        {
            return value + 1;
        }
        return value + 2;
    }

    public static void Main()
    {
        int score = Score(5);
        string label = "small";
        if (Score(8) > 7)
        {
            label = "big";
        }
        Console.WriteLine("cs_branch " + label + " " + score);
    }
}
